projects/js-packages/publicize-components/src/components/form/index.tsx

```diff
@@ -42,38 +43,28 @@ export default function PublicizeForm() {
 
 	const showSharePostForm =
 		isPublicizeEnabled &&
+		! isPublicizeDisabledBySitePlan &&
 		( hasEnabledConnections ||
 			// We show the form if there is any attached media or validation errors to let the user
 			// fix the issues with uploading an image.
 			attachedMedia.length > 0 ||
 			( Object.keys( validationErrors ).length !== 0 && ! isConvertible ) );
 
-	const Wrapper = isPublicizeDisabledBySitePlan ? Disabled : Fragment;
```

Comment:
Is this intended? Now if republicize is not supported, on shared posts we still render the toggleable connectionlist and PreviewPostsTrigger/SocialPostModal as active, but it's not working properly because they cannot share